packit/distgit.py

```diff
@@ -406,6 +408,54 @@ def build(
         )
         pkg_tool.build(scratch=scratch, nowait=nowait, koji_target=koji_target)
 
+    def get_latest_build_in_tag(self, downstream_package_name, dist_git_branch):
+        """Query Koji for the latest build of a package in a tag.
+
+        Do not use bodhi_client.latest_builds anymore,
+        it tooks to much time, it is making multiple
+        queries (more than we need) to Koji,
+        and often it timeouts.
```

Comment:
It's welcome to have this in the commit message, but I'd not put it in the docstring.